docker-compose.dev.yml

```diff
@@ -41,6 +61,42 @@ services:
         target: /etc/config/kratos
     networks:
       - intranet
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.services.kratos-public.loadbalancer.server.port=4433"
+      - "traefik.http.services.kratos-admin.loadbalancer.server.port=4434"
+
+      # Middleware to rewrite the webauthn.js path
+      - "traefik.http.middlewares.replace-webauthn-path.replacePath.path=/.well-known/ory/webauthn.js"
+      
+      # Route: Serve /.well-known/webauthn.js
+      - "traefik.http.routers.kratos-well-known.rule=Path(`/.well-known/webauthn.js`)"
+      - "traefik.http.routers.kratos-well-known.entryPoints=web"
+      - "traefik.http.routers.kratos-well-known.middlewares=replace-webauthn-path"
+      - "traefik.http.routers.kratos-well-known.service=kratos-public"
+
+      # Public routes
+      - "traefik.http.routers.kratos-oidc-callback.rule=PathPrefix(`/self-service/methods/oidc/callback`)"
+      - "traefik.http.routers.kratos-oidc-callback.entryPoints=web"
+      - "traefik.http.routers.kratos-oidc-callback.service=kratos-public"
+
+      - "traefik.http.routers.kratos-schemas.rule=PathPrefix(`/schemas`)"
+      - "traefik.http.routers.kratos-schemas.entryPoints=web"
+      - "traefik.http.routers.kratos-schemas.service=kratos-public"
+
+      # Admin routes
+      - "traefik.http.routers.kratos-admin-identities.rule=PathPrefix(`/admin/identities`)"
+      - "traefik.http.routers.kratos-admin-identities.entryPoints=web"
+      - "traefik.http.routers.kratos-admin-identities.service=kratos-admin"
+
+      - "traefik.http.routers.kratos-admin-recovery.rule=PathPrefix(`/admin/recovery`)"
+      - "traefik.http.routers.kratos-admin-recovery.entryPoints=web"
+      - "traefik.http.routers.kratos-admin-recovery.service=kratos-admin"
+
+      - "traefik.http.routers.kratos-admin-sessions.rule=PathPrefix(`/admin/sessions`)"
+      - "traefik.http.routers.kratos-admin-sessions.entryPoints=web"
+      - "traefik.http.routers.kratos-admin-sessions.service=kratos-admin"
```

Comment:
question: why do we need to expose these?

Reply:
In juju these rules are used by different traefik's, so I don't think it's needed here. But let's leave it as it is.